pkgs/tools/audio/picotts/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, libtool, popt }:
+
+stdenv.mkDerivation {
+  name = "picotts-2018-10-19";
```

```suggestion
  name = "picotts-unstable-2018-10-19";
```

https://nixos.org/nixpkgs/manual/#sec-package-naming
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".